### PR TITLE
PS-269: Fix upgrade_orphan, sdi, dd_upgrade_test, order_by_limit (8.0)

### PR DESCRIPTION
**Pull request by inikep**
1. Remove files left after `innodb.upgrade_orphan`
2. Re-record `innodb.sdi`, `main.dd_upgrade_test, `main.order_by_limit`

### CODE REVIEW

**Comment by inikep** [comment]
Tested at https://ps.cd.percona.com/view/8.0/job/percona-server-8.0-param/122/

File: mysql-test/std_data/dd/sdi/upgrade/mysql.json [hunk 1]
```diff
@@ -23057,6 +23057,7 @@
         "schema_ref": "mysql",
         "se_private_id":NNN,
         "engine": "InnoDB",
+        "last_checked_for_upgrade_version_id": X,
```
Comment (laurynas-biveinis) [inline mysql-test/std_data/dd/sdi/upgrade/mysql.json:23060]: Is it upstream, or if it is not upstream, why we are different?

File: mysql-test/std_data/dd/sdi/upgrade/mysql.json [hunk 2]
```diff
@@ -23361,6 +23362,7 @@
...
```
Comment (laurynas-biveinis) [inline mysql-test/std_data/dd/sdi/upgrade/mysql.json:23365]: Is it upstream, or if it is not upstream, why we are different?

File: mysql-test/suite/innodb/t/upgrade_orphan.test [hunk 3]
```diff
@@ -148,3 +148,5 @@ let $restart_parameters =;
 --source include/restart_mysqld.inc
 --force-rmdir $MYSQLD_DATADIR1
 --remove_file $MYSQL_TMP_DIR/data57.zip
+--remove_file $MYSQLD_LOG
```
Comment (laurynas-biveinis) [inline mysql-test/suite/innodb/t/upgrade_orphan.test:151]: Is it upstream, or if it is not upstream, why we are different?

Comment (inikep) [reply mysql-test/suite/innodb/t/upgrade_orphan.test:151]: This is upstream 8.0.13 issue

File: mysql-test/std_data/dd/sdi/upgrade/mysql.json (hunk 1 above)
Comment (inikep) [reply mysql-test/std_data/dd/sdi/upgrade/mysql.json:23060]: This part is added by Satya https://github.com/percona/percona-server/commit/14e2a96c6954433970ce15b613dbbe85fb7239f0#diff-68a5096de97c85497eb7bb7ef339437b
but `last_checked_for_upgrade_version_id` was added in upstream 8.0.13

File: mysql-test/std_data/dd/sdi/upgrade/mysql.json (hunk 2 above)
Comment (inikep) [reply mysql-test/std_data/dd/sdi/upgrade/mysql.json:23365]: as above

File: mysql-test/suite/innodb/t/upgrade_orphan.test (hunk 3 above)
Comment (laurynas-biveinis) [reply mysql-test/suite/innodb/t/upgrade_orphan.test:151]: Ack, please report to them

**Review by laurynas-biveinis — APPROVED** [review]

File: mysql-test/suite/innodb/t/upgrade_orphan.test (hunk 3 above)
Comment (inikep) [reply mysql-test/suite/innodb/t/upgrade_orphan.test:151]: Done at https://bugs.mysql.com/bug.php?id=93686

Comment (laurynas-biveinis) [reply mysql-test/suite/innodb/t/upgrade_orphan.test:151]: Thanks, please copy to JIRA too

**Pull request merged**